packages/strapi-design-system/codemod/playwright_runner_test_files.js

```diff
@@ -0,0 +1,55 @@
+// jscodeshift can take a parser, like "babel", "babylon", "flow", "ts", or "tsx"
+// Read more: https://github.com/facebook/jscodeshift#parser
+export const parser = 'babel';
+
+// Press ctrl+space for code completion
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
```

Comment:
Can we avoid naming constants as `j,s,n` ? 😄

Reply:
Nice catch, it is the default [ast-explorer](https://astexplorer.net/) convention naming (same as [facebook/jscodeshift](https://github.com/facebook/jscodeshift#the-jscodeshift-api)). But you are right, I will rename those constants.

Reply:
If it's a convention then you can leave it as is. Thanks for the clarification